src/handlers/event.ts

```diff
@@ -85,20 +88,30 @@ export async function handleEvent(definition: EventHandler, request: RawRequest,
     return handleRawEvent(definition, request, context)
 }
 
+export async function createLambdaContext(
+    definition: EventHandler<never, never>,
+    context: Context,
+    services: Promise<never> | undefined,
+    config: Promise<never> | undefined
+): Promise<LambdaContext<never, never>> {
+    return {
+        logger: globalLogger,
+        metrics: globalMetrics,
+        tracer: globalTracer,
+        isSensitive: definition.isSensitive ?? false,
+        requestId: (requestIdGenerator === 'uuid' ? randomUUID : randomUUID)(),
+        raw: context,
+        services: (await services) as never,
+        config: (await config) as never,
+    }
+}
+
 export function eventHandler<H extends EventHandler>(definition: H): AWSLambdaHandler {
     const config = isFunction(definition.config) ? Promise.resolve(definition.config()) : definition.config
     async function handler(request: RawRequest, context: Context): Promise<RawResponse> {
```

Comment:
it would be great if we can somehow inject the ID all the way at the beginning of this handler